### PR TITLE
Iac 721 move build process to GitHub actions

### PR DESCRIPTION
**Pull request by n-vodenicharski**
### Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added relevant error handling and logging messages to help troubleshooting
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have updated CHANGELOG.md with a short summary of the changes introduced
- [ ] I have tested against live environment, if applicable
- [ ] I have added relevant usage information (As-built)
- [ ] Any structure and/or content vRA-NG improvements are synchronized with vra-ng and ts-vra-ng archetypes
- [ ] Every new or updated Installer property is documented in docs/archive/doc/markdown/use-bundle-installer.md
- [ ] Dependencies in pom.xml are up-to-date
- [ ] My changes have been rebased and squashed to the minimal number of relevant commits
- [ ] My changes have a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### CODE REVIEW

**Review by Michaelpalacce — COMMENTED** [review]
Can we push SNAPSHOT builds to maven central without any constraints?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ name: Build Project
 on:
   push:
     branches:
-      - '**' # trigger the workflow on any branch push event
+      - "**" # trigger the workflow on any branch push event
     paths:
-      - '**' # trigger the workflow on any file change
+      - "**" # trigger the workflow on any file change
 
 jobs:
```
[the remark this answers is not in the document]
Comment (n-vodenicharski) [reply .github/workflows/build.yml:10]: done

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -24,12 +24,12 @@ jobs:
 
       - uses: actions/setup-java@v3
```
[the remark this answers is not in the document]
Comment (n-vodenicharski) [reply .github/workflows/build.yml]: done

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -4,15 +4,47 @@ on:
...
+          fi
+          
+      - name: Validate version input
+        run: |
+          if ! [[ "${{ inputs.version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Invalid version number: ${{ inputs.version }}"
+            exit 2
+          fi
+
+      - name: Validate branch
```
[the remark this answers is not in the document]
Comment (n-vodenicharski) [reply .github/workflows/release.yml]: done

**Pull request closed without merging**